src/test/java/org/openmrs/contrib/qaframework/automation/InpatienttSteps.java

```diff
@@ -0,0 +1,100 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.uitestframework.test.TestData;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class InpatienttSteps extends Steps {
+
+	private TestData.PatientInfo testPatient;
+
+	@Before(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void visitHomePage() {
+		testPatient = createTestPatient();
+		initiateWithLogin();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void destroy() {
+		deletePatient(testPatient);
+		quit();
+	}
+
+	@Given("a user clicks on Find Patient app from Home page")
+	public void goToFindPatientRecordPage() {
+		findPatientPage = homePage.goToFindPatientRecord();
+	}
+
+	@Then("the system loads find patient record page")
+	public void systemloadsFindPatientRecordPage() {
+		assertTrue(textExists("Find Patient Record"));
+	}
+
+	@When("a user searches an existing patient from find patient record page")
+	public void searchExistingPatient() {
+		findPatientPage.enterPatient(testPatient.identifier);
+		findPatientPage.waitForPageToLoad();
+		dashboardPage = findPatientPage.clickOnFirstPatient();
+	}
+
+	@And("a user starts visit from the patient dashboard page")
+	public void clickOnStartVisit() {
+		visitsDashboardPage = dashboardPage.startVisit();
+	}
+
+	@Then("the system loads patient visit dashboard page")
+	public void systemloadsPatientVisitDashboardPage() {
+		assertTrue(visitsDashboardPage.containsText("Visits"));
+	}
+
+	@When("a user clicks on Admit to Inpatient button and selects location")
+	public void admissionEncounter() {
```

Comment:
The new naming convention looks different from what you had before and else where. The original name explicitly specified what was to be done. For instance, `clickOnStartVisit, searchExistingPatient,  goToFindPatientRecordPage `. I find the older convention clearer and more explicit about what is to be done than `admissionEncounter, transferEncounter , dischargeEncounter `

Reply:
The commit with id `317b2b1` explains the response to the review  [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/81#discussion_r695753689).